src/index.html

```diff
@@ -5,9 +5,51 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__prduct-image"
+        src="images/imac.jpeg"
+        alt="product image"
+      >
+
+      <p class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__review">
+        <div class="card__stars card__stars--4">
+          <p class="card__star"></p>
+          <p class="card__star"></p>
+          <p class="card__star"></p>
+          <p class="card__star"></p>
+          <p class="card__star"></p>
+        </div>
+
+        <p class="card__review-amount">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class = "card__lable-price">Price:</p>
+        <p class = "card__actual-price">$2,199</p>
+      </div>
+
+      <a
+        href="#store"
+        data-qa="hover"
+        class="card__buy-button"
+      >buy
+      </a>
+
+    </div>
```

Comment:
```suggestion
      <a
        href="#store"
        data-qa="hover"
        class="card__buy-button"
      >
        buy
      </a>
    </div>
```
[CODE STYLE] - Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements